pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -328,162 +332,20 @@ public boolean getTlsEnabled() {
     }
 
 
-    @Data
```

Comment:
Removing configuration is usually treated as a breaking change. Can we keep the old configuration settings for one release but deprecate them? Then after a few more releases, we can remove the settings. Otherwise, it is painful for people to upgrade a cluster.

Reply:
@sijie I didn't remove them, I moved them done to here:

https://github.com/apache/pulsar/pull/5463/files#diff-a26c05f28ea3e67193e9ad83c63f687fR435

Reply:
There is also a test to make sure BC is preserved:

https://github.com/apache/pulsar/pull/5463/files#diff-7eb172c16a86f29eeec8f23f17993cdcR829